docs/eventing/sources/README.md

```diff
@@ -2,62 +2,69 @@
 This is a generated file and should not be changed manually. All changes should follow the
 procedure:
 
-1. Update the information in [`sources.yaml`](./sources.yaml).
+1. Update the information in [`sources.yaml`](sources.yaml).
```

Comment:
i see this is only a comment and does not render in GitHub nor the website (so it wont break anything). 
But FYI, due to the Hugo build, all links should be relative and any link to files within the same folder require the " ./ " prefix. 
(i will capture all these requirements in the docs contributor guide and push that out soon)

Reply:
ok, I fixed the ReadmeTemplate and reran.

Reply:
ok, I fixed the template the generator uses.
@RichieEscarez PTAL